main.tf

```diff
@@ -118,3 +119,13 @@ resource "aws_db_instance" "rds" {
     infrastructure-support = "${var.infrastructure-support}"
   }
 }
+
+resource "aws_db_parameter_group" "custom_parameters" {
+  name   = "${local.db_name}"
```

Comment:
This should be `"${local.identifier}"` as with the instance above, so that we can easily match parameters to the RDS instances.

Are there any other options we might want to configure in the future? Perhaps we should have "global" parameter groups we can reuse, in order to avoid hitting and having to raise service limits (50 parameter groups by default).

Reply:
This is a good point about the limit.  With this solution each RDS instance will create a parameter group.   Just looked it up, we are also limited to 40 instances per account-region.  So I guess we'd hit that limit first.  Either way we'd have to ask AWS to raise the limits for both.

Reply:
For now, I think I'd like to merge this and we can decide on whether we want to change the behaviour in the future.

It seems `parameter_group_name` [does not force a new resource](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_db_instance.go#L284-L288), however, it [would require a reboot](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithParamGroups.html). This might complicate rolling out an update but it's not as bad as the equivalent with subnet groups.